examples/regression/1-quickstart/plot_heteroscedastic_1d_data.py

```diff
@@ -0,0 +1,204 @@
+"""
+==========================================================
+Estimate the prediction intervals of 1D heteroscedastic data
+==========================================================
+
+:class:`mapie.regression.MapieRegressor` and
+`mapie.quantile_regression.MapieQuantileRegressor` is used to estimate
+the prediction intervals of 1D heteroscedastic data using
+different strategies. The latter class should provide the same
+coverage for a lower width of intervals.
```

Comment:
because it adapts the prediction intervals to the local heteroscedastic noise.

Reply:
done